### ISSUE TITLE
why there is such big difference between total gain and total loss change

### ISSUE THREAD

**Issue by Zt-Wan** [dmlc/xgboost#9196]
I have trained a model using subsample = 1, use get_score(importance_type=='total_gain') to get total gain for each feature and then sum them up. Then I calculate the difference between initial total loss and final total loss with square loss function. But I find that the sum of total gain is much larger than the total loss change. 
From my understanding, when I split a node into two part, the gain equals how much the loss decrease.
How should I interpret this difference and why this happen ?
